service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderController.java

```diff
@@ -113,6 +119,15 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
+        if (caseData.getCaseManagementOrder() == null || !caseData.getCaseManagementOrder().isInJudgeReview()) {
+            // CMO created via placeholder state
+            caseDetails.getData().remove(CASE_MANAGEMENT_ORDER_JUDICIARY.getKey());
```

Comment:
Even the comment above doesn't clarify why it needs to be removed 🤔 . Why we need to remove it?

Reply:
Yes I’d agree comment could be better. We remove it because we do not want the caseProgessionService to attempt to progress this order. I thought it is better just to remove it from caseDetails before it can be actioned any further.

Reply:
I've looked into removing this and it causes a null pointer exception, as the judge is on the placeholder page and progresses through that event without any of the order detail fields

Reply:
<img width="1036" alt="Screenshot 2020-01-03 at 11 11 04" src="https://user-images.githubusercontent.com/8612694/71720467-c7d66c80-2e19-11ea-8bc4-db8cc2c0cad2.png">